api-admin/src/main/resources/api/openapi.yml

```diff
@@ -124,11 +156,11 @@ paths:
         - { name: stableId, in: path, required: true, schema: { type: string } }
       requestBody:
         required: true
-        content: { application/json: { schema: {$ref: '#/components/schemas/VersionedFormDto' } } }
+        content: { application/json: { schema: {type: object} } }
```

Comment:
Are you feeling that maintaining openapi schema definitions is too much effort for the value? Or is this endpoint shifting to be more open-ended such that we can't define a concrete schema?

Reply:
Yeah, I don't think we're getting any value out of the schema definitions, and I find they are a huge pain to write (lack of good IDE autocomplete, having to recompile the project to see if it matches your controller).  And the reason I reverted this one is that ConsentForm and Survey have now diverged (survey has AnswerMappings and a footer property), and so it didn't make sense to have a shared DTO